### PR TITLE
Include forking the repo

### LINKED ISSUE #1007: Contributing guide should state forking the repo
In [this section](https://github.com/ploomber/ploomber/blob/master/CONTRIBUTING.md#setup-with-conda), we state cloning the repo, prior to that the user should fork the repo into their accounts and only then fork that repo.

### PR DESCRIPTION
**Pull request by sokratisvas**
## Describe your changes
Updated CONTRIBUTING.md by adding "fork this repo" step.
## Issue ticket number and link
Closes #1007

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] I have added thorough tests (when necessary).
- [x] I have added the right documentation (when needed). Product update? If yes, write one line about this update.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ conda config --add channels conda-forge
 
 Once you have conda ready:
 
+Fork this repository to your own GitHub account.
```
Comment (idomic) [inline CONTRIBUTING.md:27]: There's a command you can specify here that forks before the git clone

**Review by idomic — COMMENTED** [review]
Small fix

File: CONTRIBUTING.md (hunk 1 above)
Comment (edublancas) [reply CONTRIBUTING.md:27]: maybe a screenshot is better? (showing the fork button) i'd bet not many people use `gh`'s CLI

Comment (sokratisvas) [reply CONTRIBUTING.md:27]: Added a screenshot of the fork button.

**Comment by idomic** [comment]
Closing as already merged, feel free to take more issues!
Thanks for contributing.

**Pull request closed without merging**